### DISCUSSION TITLE
Help with Anime-DL script

### DISCUSSION THREAD

**Discussion by shaiyaofolena** [community/community#116669] [Programming Help]
hello everyone, 
I followed the tutorial for this script https://github.com/Xonshiz/anime-dl?tab=readme-ov-file but I encountered an error message with the command "python pip install - r requirements.txt" PowerShell says "PS F:\Script\anime-dl-master\anime_dl> python pip install -r requirements.txt
C:\Users\Olivia\AppData\Local\Programs\Python\Python312\python.exe: Unable to open file 'F:\ \Script\\anime-dl-master\ \anime_dl\ \pip': [Errno 2 ] No file or directory of that name
PS F:\Script\anime-dl-master\anime_dl>"

I must surely be missing something basic ... I downloaded all the dependencies, could you help me?

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @shaiyaofolena!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to Conversations related to getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by shaiyaofolena** [reply]
Thank you !
